### ISSUE TITLE
Take RSSI measurements into account for calculating distance to the device and ensure correct display of it 

### ISSUE THREAD

**Issue by Meisterlu** [amosproj/amos2022ss05-find-my-hearing-aid#199]
## User story
1. As a user 
2. I want / need have a correct measurement of the distance to my device I am searching for
3. So that I have a great expereince when using the app

## Further information
- based on 

## Acceptance criteria
- [ ] include measurement that calculates distance in meters based on RSSI value taking into account Leos Tests
- [ ] Display distance to the device based on the calculation

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by codingwithelijah** [comment]
@Chippelius: Could you give us a quick update regarding the progress of this issue? Thanks in advance :)

**Comment by Chippelius** [comment]
Please have a look: https://github.com/amosproj/amos2022ss05-find-my-hearing-aid/blob/main/Documentation/test_arduino_devices/test_results_leo.md
